src/Task/Base/Exec.php

```diff
@@ -67,6 +67,16 @@ class Exec extends BaseTask implements CommandInterface, PrintedInterface, Simul
     protected $process;
 
     /**
+     * @var resource|string
+     */
+    protected $input;
+
+    /**
+     * @var boolean
+     */
+    protected $interactive;
```

Comment:
I think that the default value should remain unset, to differentiate between unset and calling `->input(false);`

Then:

```
if (!isset($this->interactive) && function_exists('posix_isatty')) {
    $this->interactive(posix_isatty(STDOUT));
}
```